### ISSUE TITLE
Use https instead of git URL for SCM connection

### ISSUE THREAD

**Issue by kthoms** [eclipse/xtext#1652]
all gradle scm stuff should contain

```
connection = "scm:git:https://github.com/eclipse/${rootProject.name}.git
```

**Comment by cdietrich** [comment]
@kthoms this affects all gradle stuff. not the bom only

**Comment by cdietrich** [comment]
TODO find solution for eclipseSourceReferences

**Comment by cdietrich** [comment]
as of now looks good

![Bildschirmfoto 2020-12-04 um 08 56 57](https://user-images.githubusercontent.com/238736/101137199-bb045f00-360e-11eb-847a-b4fa1fef4d20.png)

**Comment by cdietrich** [comment]
fixed in 2.25
